algrtm/src/components/PianoRollDescription.js

```diff
@@ -0,0 +1,34 @@
+import Image from "next/image";
+import React from "react";
+
+const PianoRollDescription = () => {
+  return (
+    <section className="flex flex-col items-center justify-center w-full px-[12.62rem] mt-[3.94rem]">
```

Comment:
Small detail, we can directly convert padding from `202px` to `12.625rem` so `px-[12.625rem]` should work here